build.sbt

```diff
@@ -46,6 +46,25 @@ inThisBuild(Def.settings(
   scalafixScalaBinaryVersion := scalaBinaryVersion.value,
 ))
 
+// When this is updated the set of modules in ActorSystem.allModules should also be updated
```

Comment:
```suggestion
```

Reply:
I accepted your suggestion and then realized the comment is required (but was wrong). It should read `In Http.allModules`.